src/Items/ItemFishingRod.h

```diff
@@ -100,26 +100,37 @@ class cItemFishingRodHandler :
 			}
 			else if (FloaterInfo.CanPickup())
 			{
+				UInt32 LotSLevel = a_Player->GetEquippedItem().m_Enchantments.GetLevel(cEnchantments::enchLuckOfTheSea);
+
+				// Chances for getting an item from the category for each level of Luck of the Sea (0 - 3)
+				const int TreasureChances[] = {50, 71, 92, 113};  // 5% | 7.1% | 9.2% | 11.3%
+				const int JunkChances[] = {100, 81, 61, 42};  // 10% | 8.1% | 6.1% | 4.2%
```

Comment:
Potential buffer overrun if a player cheats in a fishing rod with LotS X.

Reply:
Does the chance for treasure increase when X > 3?

Reply:
I can't find anything on the wiki.  I'm sure it would be okay to just treat anything >3 the same as level 3 though.